### DISCUSSION TITLE
How can I find my wallet ？

### DISCUSSION THREAD

**Discussion by yhhwpp** [Chia-Network/chia-blockchain#7817] [Support]
I forget the mnemonic words due to  computer accident，only  ` mainnet` directory  was left,  How can I find my wallet ?

![image](https://user-images.githubusercontent.com/7929656/127093649-6a15ef6c-bd24-4801-858a-617e307bd0e8.png)

**Comment by loppefaaret** [comment]
Turning this into a support discussion

**Reply by loppefaaret** [reply]
I'm sorry to say, but if you do not have the 24 word mnemonic, it is going to be impossible to recover your 24 words mnemonics, as far as i know.

**Reply by yhhwpp** [reply]
ok
